app/src/main/java/xyz/whereuat/whereuat/LoginActivity.java

```diff
@@ -185,12 +197,74 @@ public void createNewAccount(View v) {
      * @return returns true if the phone number is a valid form
      */
     private boolean isValidPhoneForm(String phone_number) {
-        String pattern = "\\+1\\d{10}";
+        String pattern = "\\d{6}-\\d{4}";
         Pattern r = Pattern.compile(pattern);
         Matcher m = r.matcher(phone_number);
         return m.find();
     }
 
+    private void setAreaEditListeners() {
+        mAreaEdit.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence s, int start, int count, int after) {
```

Comment:
curlies on same line? easy to see that the body is empty and we reduce line length